### PR TITLE
Ceph monitoring improvement

### PR DESCRIPTION
**Pull request by Xiaohua-Shen**
1. add ceph cluster usage check
2. modify ceph health check to escalate some warning situations

### CODE REVIEW

**Comment by Xiaohua-Shen** [comment]
This PR has dependency on usula-monitoring's PR blueboxgroup/ursula-monitoring#95/, so no need to merge it now.
we can review these two PR together, and merge this PR after ursula-monitoring's PR

**Comment by j2sol** [comment]
Please rebase on master.

**Comment by Xiaohua-Shen** [comment]
I executed rebase and it's up-to-date.   ursula-monitoring related PR should be review and merge first, then this PR.

**Comment by knandya** [comment]
LGTM

File: roles/ceph-monitor/tasks/monitoring.yml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 ---
 - name: ceph status check
   sensu_check: name=check-ceph-status plugin=check-ceph.rb
-               use_sudo=true args='--criticality {{ ceph.monitoring.sensu_checks.check_ceph_status.criticality }}'
+               use_sudo=true args='--criticality {{ ceph.monitoring.sensu_checks.check_ceph_status.criticality }} -e "osds are down,near full osd,clock skew,mons down" -o -f -p'
```
Comment (j2sol) [inline roles/ceph-monitor/tasks/monitoring.yml]: The -e arguments should be a variable defined in the role defaults. That way environments can override what they care about.

Comment (Xiaohua-Shen) [inline roles/ceph-monitor/tasks/monitoring.yml]: I already add one variable defined in the role defaults, please review again.

Comment (Xiaohua-Shen) [inline roles/ceph-monitor/tasks/monitoring.yml]: one question: if add this variable defined in the role default, then all existing blue box env file should be modified too

Comment (j2sol) [inline roles/ceph-monitor/tasks/monitoring.yml]: No, the existing env files shouldn't have to be modified. This is a new variable, and the default is fine. We'd only have to modify the env files if we want something other than the default.

**Comment by j2sol** [comment]
ok to test

**Pull request merged**